part-spec/ic_io.json

```diff
@@ -783,6 +787,164 @@
                     }
                 }
             }
+        },
+        {
+            "title": "multiplexer",
+            "type": "object",
+            "required": [
+                "partType",
+                "manufacturer",
+                "mpn",
```

Comment:
Replace all these properties (partType, manufacturing, mpn,...) by "componentID" object introduced in rev0.5